### ISSUE TITLE
xhair before xhair-mode makes xhair-mode non-persistent

### ISSUE THREAD

**Issue by ecbrown** [Boruch-Baum/emacs-xhair#1]
Hello Boruch,

I have tried this package and I seem to observe that:

M-x xhair 

called before 

M-x xhair-mode

renders it such that xhair-mode is non-persistent. If I call xhair-mode first, it works as expected.

GNU Guix with Emacs 27.2, only (require 'xhair) and emacs -nw -Q

Best regard,
Eric

**Comment by Boruch-Baum** [comment]
Verified. A hook was being applied globally instead of buffer-local.
I've pushed a commit that should clear up the issue. In order to benefit
from the fix without needing to restart Emacs, evaluate the following:

    (remove-hook 'pre-command-hook #'xhair-unhighlight)

Then just re-evaluate the elisp buffer, or if you're stingy just
evaluating the corrected function xhair-highlight should be sufficient.

Thanks for the report.
